app/helpers/media_helper.rb

```diff
@@ -149,4 +149,14 @@ def external_link_description_not_empty(medium)
     # link url itself.
     medium.external_link_description.presence || medium.external_reference_link
   end
+
+  def video_link_timed(medium_id, timestamp)
+    play_path = Rails.application.routes.url_helpers.play_medium_path(medium_id)
```

Comment:
It is not necessary invoke `Rails.application.routes.url_helpers` if you are already inside a helper, `play_medium_path` should be sufficient. Also, it would be more in the Rails way to give `medium` as a parameter and use `play_medium_path(medium)`.